### PR TITLE
[DEVOPS-1410] ElementIsVisible() method override for elementExists()

### PR DESCRIPTION
**Pull request by AndreKrebs**

### CODE REVIEW

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java [hunk 1]
```diff
@@ -96,11 +96,11 @@ protected void scrollToElement(WebElement element) {
     }
 
     protected void waitTextInElement(WebElement element, String expected) {
-        waitAllConditions(element, elementIsVisible(), elementTextContains(expected));
+        waitAllConditions(element, elementExist(), elementTextContains(expected));
     }
 
     protected void waitUntilElementExistsWithText(By by, String valueToSelect) {
-        waitAllConditions(by, elementIsVisible(), elementTextIs(valueToSelect));
+        waitAllConditions(by, elementExist(), elementTextIs(valueToSelect));
```
Comment (jhonypg) [inline application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: waitUntilElementIsVisibleExistsWithText

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java [hunk 2]
```diff
@@ -96,11 +96,11 @@ protected void scrollToElement(WebElement element) {
     }
 
     protected void waitTextInElement(WebElement element, String expected) {
-        waitAllConditions(element, elementIsVisible(), elementTextContains(expected));
+        waitAllConditions(element, elementExist(), elementTextContains(expected));
```
Comment (jhonypg) [inline application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: waitUntilElementIsVisibleExistsWithText

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java [hunk 3]
```diff
@@ -174,7 +174,7 @@ protected WebElement getElementWhenItExistsAndIsVisible(By by) {
     }
 
     protected WebElement waitUntilChildElementExists(WebElement element, By by) {
-        return waitAllConditions(element, by, elementIsVisible());
+        return waitAllConditions(element, by, elementExist());
```
Comment (jhonypg) [inline application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: waitUntilChildElementIsVisible

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java [hunk 4]
```diff
@@ -198,7 +198,7 @@ protected WebElement getChildElementWhenExists(By elementSelector, By childEleme
     }
 
     protected void waitForClick(WebElement element) {
-        waitAllConditions(element, elementIsVisible(), elementIsClickable(), elementClicked());
+        waitAllConditions(element, elementExist(), elementIsClickable(), elementClicked());
```
Comment (jhonypg) [inline application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: Keep IsVisible

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java [hunk 5]
```diff
@@ -213,7 +213,7 @@ protected void waitForClickHoldingAKey(WebElement element, Keys keyHolding) {
     }
 
     protected void waitForClick(By by) {
-        waitAllConditions(by, elementIsVisible(), elementIsClickable(), elementClicked());
+        waitAllConditions(by, elementExist(), elementIsClickable(), elementClicked());
```
Comment (jhonypg) [inline application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: Keep IsVisible

Comment (AndreKrebs) [reply application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: ok

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java (hunk 4 above)
Comment (AndreKrebs) [reply application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: ok

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java (hunk 3 above)
Comment (AndreKrebs) [reply application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: ok

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java (hunk 2 above)
Comment (AndreKrebs) [reply application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: ok

File: application/src/test/java/objective/taskboard/it/AbstractUiFragment.java (hunk 1 above)
Comment (AndreKrebs) [reply application/src/test/java/objective/taskboard/it/AbstractUiFragment.java]: ok

**Pull request merged**